packages/dashboards/addon/routes/dashboards/dashboard/view.js

```diff
@@ -150,11 +147,11 @@ export default Route.extend({
    * @param {Object} controller
    * @param {Boolean} isExiting
    */
-  resetController(controller, isExiting /*transition*/) {
+  deactivate() {
     this._super(...arguments);
 
-    if (isExiting) {
-      controller.resetModel();
-    }
+    this.controllerFor('dashboards.dashboard.view').set('filters', null);
```

Comment:
Can this work
`this.controller`

 https://api.emberjs.com/ember/3.12/classes/Route/properties/controller?anchor=controller